app/components/Navbar/Navbar.tsx

```diff
@@ -1,11 +1,7 @@
-import Link from "next/link"
```

Comment:
This commit should be named as chore: support button incapsulated
chore - for update that doesn't affect on user